<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QDialog" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeBack">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>117</y>
     <width>416</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font:60px;
qproperty-wordWrap: true;
font-family: fontSpaceGrotesk;</string>
   </property>
   <property name="text">
    <string>Welcome back, </string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>117</y>
     <width>511</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font:60px;
qproperty-wordWrap: true;
font-family: fontSpaceGrotesk;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="yourSets">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>210</y>
     <width>202</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font:40px;
qproperty-wordWrap: true;
font-family: fontSpaceGrotesk;</string>
   </property>
   <property name="text">
    <string>Your sets:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yourCustomSets">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>463</y>
     <width>374</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font:40px;
qproperty-wordWrap: true;
font-family: fontSpaceGrotesk;</string>
   </property>
   <property name="text">
    <string>Your custom sets:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set1">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>264</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set2">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>264</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set3">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>264</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="set4">
   <property name="geometry">
    <rect>
     <x>1102</x>
     <y>264</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="customSet1">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>513</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="customSet2">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>513</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="customSet3">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>513</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="customSet4">
   <property name="geometry">
    <rect>
     <x>1101</x>
     <y>513</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="leftArrowSets">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>326</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8E8E8E;
font:40px;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightArrowSets">
   <property name="geometry">
    <rect>
     <x>1441</x>
     <y>326</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8E8E8E;
font:40px;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftArrowCustom">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>575</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8E8E8E;
font:40px;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightArrowCustom">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>575</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8E8E8E;
font:40px;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createNewSet">
   <property name="geometry">
    <rect>
     <x>1384</x>
     <y>460</y>
     <width>57</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8E8E8E;
font:60px;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewSet">
   <property name="geometry">
    <rect>
     <x>1384</x>
     <y>209</y>
     <width>57</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8E8E8E;
font:60px;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
